noetic/distribution.yaml

```diff
@@ -521,6 +521,13 @@ repositories:
       url: https://github.com/ros-perception/calibration.git
       version: noetic-devel
     status: maintained
+  camera_aravis:
+    source:
+      test_pull_requests: true
```

Comment:
did you follow the steps detailed here http://wiki.ros.org/buildfarm/Pull%20request%20testing#Link_Jenkins_and_your_GitHub_repository?